### DISCUSSION TITLE
How to Ctrl+Shift+U unicode?

### DISCUSSION THREAD

**Discussion by weedeater64** [qutebrowser/qutebrowser#7548] [Q&A]
Title.

**Comment by mschilli87** [comment]
OS? :wink:

**Reply by weedeater64** [reply]
Linux Mint 20 with Xfce. Installed qutebrowser from repos.

**Comment by mschilli87** [comment]
What do you get from `Ctrl+Shift+U`? Do you see the underlined u at all? Is `XMODIFIERS` set? Does it work in other app? Incl. other Qt apps? Did you try passthrrough mode (Ctrl+v)?
